src/stick.jl

```diff
@@ -83,34 +83,4 @@ function readstick()
     end
 end
 
-"""
-    sticktask()
```

Comment:
Done. 
`Base.depwarn` does not seem to work on the ARM build provided on julialang.org
Probably works on the `apt-build` version since it crashes whenever there is a warning and it did.

Reply:
@simonbyrne I made the changes. I don't know whether you got the notification or not. I have no idea about how busy you are. Let me know should I DM you/tag you after I make changes. I wouldn't want to be annoying.